app/models/plan.rb

```diff
@@ -39,7 +39,7 @@ def deletable?
   end
 
   def has_trial?
-    trial_period.present?
+    trial_period.present? && trial_period.positive?
```

Comment:
Don't we want to default `trial_period` to `0.0` instead of managing `nil` values?

Reply:
Hmm, good question :thinking:  (however the issue would be the same)